### PR TITLE
Add Outage

### PR DESCRIPTION
**Pull request by T0biii**
addes the Cert Issoue Outage.
took the time from here: https://chat.ffmuc.net/freifunk/pl/tbyoj6q8qtgwxc8k9qz6fkrqao

I take Text suggestions gladly :)

### CODE REVIEW

File: _includes/network-status.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 <br>Aktuell sind uns keine Störungen bekannt
 <br>
 <br><u><b>History:</b></u>
+<br><b>22.05.2022 23:43 - 23.05.2022 09:25</b> Ausfall einzelener FFMUC Dienste (z.B. DoT, Chat, Ticket-System, CryptPad)
 <br><b>01.10.2020 10:20 - 13:46</b> Ausfall des DoT Servers (dot.ffmuc.net)
```
[the remark this answers is not in the document]
Comment (T0biii) [reply _includes/network-status.md:9]: I would not delete historical data

[the remark this answers is not in the document]
Comment (GoliathLabs) [reply _includes/network-status.md:9]: Why? Some incidents are over two years old. We do not track every small incident and listing many (especially) old ones makes the website visitor think that we do not have a reliable infrastructure.

File: _includes/network-status.md [hunk 2]
```diff
@@ -5,6 +5,7 @@
 <br>Aktuell sind uns keine Störungen bekannt
 <br>
 <br><u><b>History:</b></u>
+<br><b>22.05.2022 23:43 - 23.05.2022 09:25</b> Ausfall einzelener FFMUC Dienste (z.B. DoT, Chat, Ticket-System, CryptPad)
```
Comment (krombel) [inline _includes/network-status.md:8]: s/einzelener/einzelner/

**Review by krombel — CHANGES_REQUESTED** [review]

File: _includes/network-status.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (T0biii) [reply _includes/network-status.md:9]: mh maybe we should than setup the status.ffmuc.net Page
Where we can easyl show if we go a Problem?

**Review by GoliathLabs — APPROVED** [review]

**Comment by GoliathLabs** [comment]
@T0biii why did you close this PR?

**Comment by T0biii** [comment]
oh wasn't on purpose

**Review by awlx — APPROVED** [review]

**Comment by T0biii** [comment]
cant merge this pr

**Pull request merged**